service/controller/clusterapi/v19/resources/app/desired.go

```diff
@@ -0,0 +1,96 @@
+package app
+
+import (
+	"context"
+	"strconv"
+
+	g8sv1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/microerror"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	cmav1alpha1 "sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
+
+	"github.com/giantswarm/cluster-operator/pkg/annotation"
+	"github.com/giantswarm/cluster-operator/pkg/label"
+	"github.com/giantswarm/cluster-operator/pkg/project"
+	pkgkey "github.com/giantswarm/cluster-operator/pkg/v19/key"
+	awskey "github.com/giantswarm/cluster-operator/service/controller/aws/v19/key"
+	azurekey "github.com/giantswarm/cluster-operator/service/controller/azure/v19/key"
```

Comment:
All these key package imports make me sad. This is all wrong from a design perspective, or at least a huge smell. I get we have to deal with this for now but these are things I want to make better in the future. I also noticed this when refactoring the other resources recently.

Reply:
Yes I don't like this either. See my other comment about the release CRs. That will move this config out of the operator so its easier to change.